addons/web/static/src/views/fields/pdf_viewer/pdf_viewer_field.js

```diff
@@ -43,7 +44,7 @@ export class PdfViewerField extends Component {
         if (!this.state.isValid || !this.props.record.data[this.props.name]) {
             return null;
         }
-        const page = this.props.record.data[`${this.props.name}_page`] || 1;
+        const page = this.props.page || this.props.record.data[`${this.props.name}_page`] || 1;
```

Comment:
You try to use a field in not common use case :/ It look strange to as a props page that is not configurable from the arch